models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql

```diff
@@ -27,16 +28,17 @@ WITH quo_evt AS (
            premiumNXM,
            scAdd,
            sumAssured,
-           '0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b' as token
+           cast('0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b' as varbinary) as token
```

Comment:
`0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b` is varbinary type.
```suggestion
           0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b as token
```